libnetwork/networkdb/cluster.go

```diff
@@ -244,7 +244,7 @@ func (nDB *NetworkDB) clusterLeave() error {
 
 func (nDB *NetworkDB) triggerFunc(stagger time.Duration, C <-chan time.Time, f func()) {
 	// Use a random stagger to avoid synchronizing
-	randStagger := time.Duration(uint64(rnd.Int63()) % uint64(stagger))
+	randStagger := time.Duration(uint64(rnd.Int63()) % uint64(stagger)) // nolint:gosec
```

Comment:
Do you recall what it was complaining about here (i suspect `rnd.Int63()` not being random?) If you do,  would be great to leave it as a comment. You can use this format:

```suggestion
	randStagger := time.Duration(uint64(rnd.Int63()) % uint64(stagger)) //nolint:gosec // some comment here what we're ignoring
```

Reply:
Ah was trying to add extra comments, didn't realize a 2nd `//` was needed.
This is do to rand use.